App/src/pages/dossier/DossierReview.tsx

```diff
@@ -107,11 +121,13 @@ export default function DossierReview() {
                 <AlertDialogOverlay>
                     <AlertDialogContent>
                         <AlertDialogHeader fontSize="lg" fontWeight="bold">
-                            Forward Dossier
+                            {currentGroup?.isFinalStage ? "Accept Dossier" : "Forward Dossier"}
                         </AlertDialogHeader>
 
                         <AlertDialogBody>
-                            Are you sure you want to forward this dossier to the next group in the approval pipeline?
+                            {currentGroup?.isFinalStage
+                                ? "Are you sure you want to accept this dossier?"
```

Comment:
There should be a space after the question mark.

Reply:
Fixed.